content/applications/hr/fleet/configuration.rst

```diff
@@ -0,0 +1,234 @@
+=============
+Configuration
+=============
+
+The following document goes through all of the configurations and settings for the *Fleet*
+application, including settings, manufacturers, models, and categories.
+
+Settings
+========
+
+To access the settings menu, go to :menuselection:`Fleet application --> Configuration -->
+Settings`. There are only two settings that need configuration, the vehicle contract alert, and
+a new vehicle request.
+
+End Date Contract Alert
+-----------------------
+
+To help ensure no contracts lapse or vehicles are left without contracted drivers, Odoo has an
+alert in place to let the person responsible for the vehicle contract know when a contract is abut
+to expire. Enter the number of days before a contract is going to expire that an alert email will be
+sent.
+
+.. note::
+   To determine who the responsible person is for a contract, open an individual contract. The
+   person listed as :guilabel:`Responsible` under the :guilabel:`Contract Information` section of
+   the contract is the person who will receive the alert.
+
+   To access all contracts, navigate to :menuselection:`Fleet app --> Fleet --> Contracts` and all
+   contracts appear in the list. Click on a :guilabel:`Contract` to view it.
+
+   An individual contract can also be found by navigating to :menuselection:`Fleet app --> Fleet -->
+   Fleet` and clicking on an individual vehicle. On the vehicle form, click the
+   :guilabel:`Contracts` smart button at the top of the page. The contract(s) associated with this
+   vehicle only appears in the list. Click on an individual contract to open it. The
+   :guilabel:`Responsible` person is listed on the contract.
+
+New Vehicle Request
+-------------------
+
+Set a limit on how many vehicle are in the fleet by setting a limit on requests. When using the
+salary configurator, employees will not be able to request a new car if there is a specific amount
+of available cars already in the fleet that do not have a current driver assigned to them. Enter the
+number limit for existing available cars in this field.
+
+.. image:: configuration/fleet-settings.png
+   :align: center
+   :alt: Settings available for the Fleet application.
+
+Manufacturers
+=============
+
+Odoo Fleet comes pre-configured with sixty-six commonly used car manufacturers in the database,
+along with their logos. To view the pre-loaded manufacturers, go to :menuselection:`Fleet app -->
+Configuration --> Manufacturers`.
+
+The list of manufacturers appear in alphabetical order. Each manufacturer's card lists how many
+specific models are configured for each particular manufacturer. Odoo comes with forty five
+pre-configured models for four major auto manufacturers.
```

Comment:
example? you talk numbers twice here and its a little confusing.

Reply:
I changed "for" to "from" and the manufacturers, so I think it's clearer now. 

Odoo comes with forty five pre-configured models from four major auto manufacturers: Audi, BMW, Mercedes, and Opel.